ocfweb/docs/docs/services/web.md

```diff
@@ -11,6 +11,49 @@ where `user` is the account name.
 Group accounts may also request a [[virtual host|doc services/vhost]] for
 another domain (e.g., group.berkeley.edu).
 
+#### Off-site hosting
+
+One question that often comes up is if the OCF is able to host websites
+from services such as Squarespace, Wix, Weebly, Tumblr, etc. Unfortunately
```

Comment:
I think it should be made clear here that we _can_ host websites _imported_ from these services, but that we don't have the same editing interfaces and stuff that they do. We also can't somehow host a website also hosted by them, that doesn't make any sense, which is more the point you made in the next paragraph.